modules/integrate-with-docker-trusted-registry-scanner.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-vulnerability-scanners.adoc
+:_module-type: PROCEDURE
+[id="integrate-with-docker-trusted-registry-scanner_{context}"]
+= Integrating with Docker Trusted Registry scanner
+
+You can integrate {product-title} with Docker Trusted Registry for scanning images.
+
+.Prerequisites
+* You need a *Username* and a *Password* for authentication with the Docker Trusted Registry.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Image Integrations* section, select *Google Container Registry*.
```

Comment:
Should this step reference Docker Trusted Registry instead of Google Container Registry?